pkg/debian/rules

```diff
@@ -38,10 +41,18 @@ override_dh_auto_install:
 	pyinstaller $(ROOT_DIR)/gatekeeper/api/src/main.py -y -F -n gk-server --distpath $(ARTIFACTS_DIR)
 	pyinstaller $(ROOT_DIR)/gatekeeper/cli/src/main.py -y -F -n gkctl --distpath $(ARTIFACTS_DIR)
 
-	# Copy all binaries and libs
+	# Copy all binaries
 	mkdir -p $(USR_BIN_DIR) $(USR_LIBS_DIR)
 	find $(ARTIFACTS_DIR) -maxdepth 1 -type f ! -name "*.*" -executable -exec cp {} $(USR_BIN_DIR) \;
 
+	# Build hyperon_das.so
+	rm -rf /tmp/das
+	git clone --depth 1 --branch das-435 https://github.com/singnet/das.git /tmp/das
```

Comment:
Is this command line correct? I'm struggling to understand why we have this arg:`--branch das-435`